catalog/async/catalog-async-data/src/main/java/org/codice/ddf/catalog/async/data/impl/LazyProcessResourceImpl.java

```diff
@@ -84,9 +92,21 @@ public LazyProcessResourceImpl(String metacardId, Supplier<Resource> resourceSup
    * <p>This will load the resource.
    */
   @Override
-  public InputStream getInputStream() throws IOException {
+  public synchronized InputStream getInputStream() throws IOException {
     loadResource();
```

Comment:
✏️ Should we move this under the `if` statement?

Reply:
❓ maybe I missed it but did you actually move `loadResource()` under the `if`?

Reply:
@shaundmorris I plan to. I was just waiting for another review so that if I needed to make any other changes I could make them all at once